extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -158,7 +170,13 @@ export class Bind {
     user().assert(this.enabled_, `Experiment "${TAG}" is disabled.`);
 
     // TODO(choumx): What if `state` contains references to globals?
-    Object.assign(this.scope_, state);
+    if (Object.keys(this.scope_).length > 0) {
```

Comment:
What if there are multiple `amp-state` elements? Perhaps this should be an input param?